shed/event_streams.py

```diff
@@ -775,9 +776,18 @@ def _start_update(self, x, who=None):
         # TODO: should we have something like event_contents for starts?
         name, docs = self.curate_streams(x, False)
         if name == 'start':
-            self.truth_value = self.predicate(docs)
+            try:
```

Comment:
This way we allow the predicate (and the `doc_contents`) the opportunity to fail gracefully.